telemetry/README.md

```diff
@@ -1,4 +1,5 @@
 # Telemetry
```

Comment:
What was this reformatted with/I want to make sure we don't get into formatting wars.